themes/font/font.theme.bash

```diff
@@ -22,20 +23,20 @@
 #
 
 SCM_NONE_CHAR=''
-SCM_THEME_PROMPT_DIRTY=" ${red}✗"
+SCM_THEME_PROMPT_DIRTY=" ${red?}✗"
 SCM_THEME_PROMPT_CLEAN=""
-SCM_THEME_PROMPT_PREFIX="${green}|"
-SCM_THEME_PROMPT_SUFFIX="${green}|"
+SCM_THEME_PROMPT_PREFIX="${green?}|"
+SCM_THEME_PROMPT_SUFFIX="${green?}|"
 SCM_GIT_SHOW_MINIMAL_INFO=true
 
 CLOCK_THEME_PROMPT_PREFIX=''
 CLOCK_THEME_PROMPT_SUFFIX=' '
 THEME_SHOW_CLOCK=false
-THEME_CLOCK_COLOR=${THEME_CLOCK_COLOR:-"$bold_blue"}
+THEME_CLOCK_COLOR=${THEME_CLOCK_COLOR:-"$bold_blue?"}
```

Comment:
This is also the same.

```suggestion
THEME_CLOCK_COLOR=${THEME_CLOCK_COLOR:-"${bold_blue?}"}
```

I haven't carefully checked, so maybe there are even more.

Reply:
I did a search for `?"}` and found only one other instance, which I also have corrected,.